storage/setup.py

```diff
@@ -31,7 +31,7 @@
 dependencies = [
     "google-auth >= 1.2.0",
     "google-cloud-core >= 1.0.3, < 2.0dev",
-    "google-resumable-media >= 0.4.0",
+    "google-resumable-media >= 0.3.1, < 0.5.0dev",
```

Comment:
This is a match to the manually released 1.19.1
We can rev this to 0.4.0 since this was an unreleased change
@tseaver

Reply:
```suggestion
    "google-resumable-media >= 0.4.0, < 0.5.0dev",
```

Reply:
ISTM that `>= 0.3.1, != 0.4.0, < 0.5dev` is the right specification, until GRMP has a new version (might be `1.0.0`?) to include deliberately:  the reality is that `0.3.1` and `0.4.1` are near identical, and the storage client works with the older one.